core/driver/sources/src/main/scala/org/virtuslab/ideprobe/ide/intellij/Display.scala

```diff
@@ -2,13 +2,8 @@ package org.virtuslab.ideprobe.ide.intellij
 
 sealed trait Display
 object Display {
-  private val Default = Native
 
-  val Mode: Display = {
-    Option(System.getenv("IDEPROBE_DISPLAY"))
-      .map(fromName)
-      .getOrElse(Default)
-  }
+  val Mode: String => Display = displayFromConfig => fromName(displayFromConfig)
```

Comment:
In fact I'd just make `fromName` public and use it in e.g. `InstalledIntelliJ`, instead of `Mode` (which is unclear now)

Reply:
Makes sense. Change for `val Mode` is not compatible anyway... Let me fix this

Reply:
Replaced `Display.Mode` with a new value: `InstalledIntelliJ.displayMode` - to avoid multiple `Display.fromName` method calls with the same input.